yandexcloud/_sdk.py

```diff
@@ -8,8 +9,8 @@
 from yandexcloud._wrappers import Wrappers
 
 
-class SDK(object):
-    def __init__(self, interceptor=None, user_agent=None, **kwargs):
+class SDK:
+    def __init__(self, interceptor=None, user_agent=None, endpoints: Optional[Dict[str, str]] = None, **kwargs):
```

Comment:
`dict[str, str] | None`, AFAIK in python 3.8 you can use new syntax

Reply:
it is not supported in python 3.8